articles/whats-wrong-with-i-dont-write-any-tests-i-am-not-a-tester.md

```diff
@@ -0,0 +1,90 @@
+---
+authorName: Peter Gfader
+authorGithubUsername: peitor
+issue: 149
+title: What’s Wrong With: “I don’t write any tests since I am not a tester”?
+---
+# {{page.title}}
+
+"Not a tester, so what are you then?" you might ask.  
+Being that offending is generally not helpful.  
+Unless you try to catch the attention as I do in this article 😉  
+Let’s digest the situation in detail.
+
+A friend of mine attended my [Scrum Developer class](https://www.scrum.org/courses/professional-scrum-developer-training) and caught fire during the "Testing" module where we talk about Code Quality, Testing, Test-First approaches, TDD and more.
+Boom! After that class he was on a mission to convince everyone that TDD is the only way to do things.
+
+The 1st day back at work he talked about improving the team and trying TDD and got the following statement from his colleague:
+Henri (made up name): "I don’t write any tests since I am not a tester".
+
+I know he handled the situation quite well, but he asked me for advice.
+
+One thing to consider is the underlying question to this, which might be:
+"How do we get people to change their behavior?"
+So here are my thoughts.
+
+## **Yourself First**   
+First of all, think about things to consider for yourself:
+* Why is the practice or tool that you are suggesting any better than the current way of doing things?
+* Can you explain the value in the proposed change?
+* Can you lead by example?
+* Do you have enough patience and skills to teach others?
+I would try to work on yourself before trying to change others.
+    
+ 
+## **Roles?**  
+I see that a lot of people are focused on their role, forgetting the bigger picture of the team and purpose of the work they are doing.
+I would ask these questions:
+* Are we one team that focuses on the Sprint Goal?
+  In a Scrum context there is no “tester”, “programmer” or “architect”, we are all professional engineers that deliver value through collaboration.
+* No matter what, do we stand together and support each other?
+* Are we doing whatever is necessary to deliver a usable product every Sprint?
+ 
+
+## **Done?**  
+Henri: "I don’t write any tests since I am not a tester".  
+Ask: "How do you know when you are done?"  
+  
+What is on our Definition of Done?
+How can we build a usable, tested and fully integrated product increment every Sprint?
+Are we doing that already? Why not?
+   
+
+## **Tests are as important as documentation**  
+Documentation is needed, and one good way to document how software **must** work are tests. I emphasize **must**, since documentation only documents how the software **might** work.  
+We learned too often that documentation gets out of sync too easily.  
+I blogged about the ["Why are *automated tests* so important?"](http://blog.gfader.com/2010/10/why-are-automated-tests-so-important.html) on my personal blog.  
```

Comment:
Instead of linking another blog here - which might not work as well in the book, I would recommend here to qucikly summarize some important arguments why developers should write automated tests (just most important ones, you could still link the blog at the end oft the article).

Reply:
Good point. I will work that into the article